src/cdk/drag-drop/drag-ref.ts

```diff
@@ -495,7 +495,7 @@ export class DragRef<T = any> {
 
   /** Handler that is invoked when the user lifts their pointer up, after initiating a drag. */
   private _pointerUp = (event: MouseEvent | TouchEvent) => {
-    if (!this.isDragging()) {
+    if (!this._dragDropRegistry.isDragging(this)) {
```

Comment:
Add a comment that explains why this is not using `this.isDragging()`?

Reply:
Done.